demosplan/DemosPlanCoreBundle/Logic/Permission/AccessControlPermissionService.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of the package demosplan.
+ *
+ * (c) 2010-present DEMOS plan GmbH, for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+namespace demosplan\DemosPlanCoreBundle\Logic\Permission;
+
+use DemosEurope\DemosplanAddon\Contracts\Entities\CustomerInterface;
+use DemosEurope\DemosplanAddon\Contracts\Entities\OrgaInterface;
+use demosplan\DemosPlanCoreBundle\Entity\Permission\AccessControlPermission;
+use demosplan\DemosPlanCoreBundle\Logic\CoreService;
+use demosplan\DemosPlanCoreBundle\Logic\User\RoleHandler;
+use demosplan\DemosPlanCoreBundle\Repository\AccessControlPermissionRepository;
+
+/**
+ * This file is part of the package demosplan.
+ *
+ * (c) 2010-present DEMOS plan GmbH, for more information see the license file.
+ *
+ * All rights reserved
+ */
+class AccessControlPermissionService extends CoreService
+{
+    public const CAN_CREATE_PROCEDURES = 'feature_admin_new_procedure';
```

Comment:
minor naming:
`CREATE_PROCEDURES_PERMISSION` (my favor)
`PERMITTED_TO_CREATE_PROCEDURES`
 or even
`PERMITTED_TO_CREATE_PROCEDURES_PERMISSION`

This way it is maybe more clear that this is a string which holds one of your permissions?! :thinking: